src/config/test_config.py

```diff
@@ -1,6 +1,6 @@
 # Please only set it to True if you are preparing a Benchmark related PR
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
-ENABLE_BENCHMARK_DEV_MODE = False
+ENABLE_BENCHMARK_DEV_MODE = True
```

Comment:
We don't need it now, while we're testing a functionality. You can set it back to False.

Reply:
Plus we changed this recently. Now we have 1 [.toml file](https://github.com/aws/deep-learning-containers/blob/master/dlc_developer_config.toml) for all the settings

Reply:
Rolled back